src/components/candidateContent/candidateInfo/CandidateInfo.jsx

```diff
@@ -27,25 +32,77 @@ const CandidateInfo = (props) => {
     return newInfo;
   };
 
+  const onButtonCancel = () => {
+    setIsEditModeOn(false);
+  };
+
+  const handleChange = (index) => (e) => {
+    console.log(`value => ${e.target.value}`);
+    const newArr = { ...newValues };
+    newArr[index] = e.target.value;
+    setNewValues(newArr);
+  };
+
   const formatedInfo = formatInfo(candidateInfo);
 
+  const onButtonEdit = () => {
+    setNewValues({ ...formatedInfo });
+    setIsEditModeOn(true);
+  };
+
+  const onButtonSave = () => {
+    console.log(`SAVE`);
+    delete newValues.fullName;
```

Comment:
Why did tou do this?

Reply:
> Why did tou do this?


Because there is no "fullname" field in the database

Reply:
And how did you get this field ?

Reply:
Create in formatInfo
newInfo.fullName = `${info.firstName} ${info.lastName}`;

Reply:
Great. Don't create extra field fullName. Just use ${info.firstName} ${info.lastName}

Reply:
So. BE doesn't have fullName, but you render tht field on edit mode. It's wrong

![image](https://user-images.githubusercontent.com/75979545/142855576-8715c9b4-33b5-42f5-9263-ce374c7f43cf.png)

Reply:
You can 1) parse fullName field, 2) ask BE to create thet field on DB (no the best option) 3)  use 2 field firstName and lastName instead of fullName